qutip/cy/cqobjevo.pyx

```diff
@@ -1,37 +1,3 @@
-#cython: language_level=3
-# distutils: language = c++
```

Comment:
This `distutils: language = c++` is also pretty odd and can hopefully go too.

Reply:
It's forced by our `setup.py` anyway, so no big deal.